build-tools/automation/azure-pipelines.yml

```diff
@@ -0,0 +1,215 @@
+# Xamarin.Android Pipeline
+
+trigger:
+  - master
+  - d16-?
+
+# Global variables
+variables:
+  BundleArtifactName: bundle
+  AutoProvisionArgs: /p:AutoProvision=True /p:AutoProvisionUsesSudo=True /p:IgnoreMaxMonoVersion=False
+
+stages:
+- stage: prepare
+  displayName: Prepare
+  variables:
+    Configuration: Release
+    MSBuildAbiArgs: >-
+      /p:AndroidSupportedTargetJitAbis=armeabi-v7a:arm64-v8a:x86:x86_64
+      /p:AndroidSupportedHostJitAbis=Darwin:mxe-Win32:mxe-Win64
+      /p:AndroidSupportedTargetAotAbis=armeabi-v7a:arm64:x86:x86_64:win-armeabi-v7a:win-arm64:win-x86:win-x86_64
+  jobs:
+  - job: create_bundle
```

Comment:
Is there a convention for job names (I assume this is a name)?  Any reason to use underscores over hyphens?

Reply:
Stage and Job names have the following limitations:  `Valid names may only contain alphanumeric characters and '_' and may not start with a number.` I haven't yet found any official documentation on pipeline naming conventions/restrictions, but I've encountered this error before:
```
Pipeline configuration error:
Stage build-mac has an invalid name. Valid names may only contain alphanumeric characters and '_' and may not start with a number.
Job first-job has an invalid name. Valid names may only contain alphanumeric characters and '_' and may not start with a number.
```